stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketConstants.java

```diff
@@ -97,6 +128,25 @@
     public static final String THE_WEBSOCKET_CONNECTION_HAS_NOT_BEEN_MADE =
             "The WebSocket connection has not been made";
 
+    public static final String CONNECTED_TO = "Connected to ";
+    public static final String STATEMENT_FOR_RECONNECT = "Maximum retry attempts but couldn't connect " +
+            "to the server: ";
+    public static final String STATEMENT_FOR_FAILOVER = "Couldn't connect to one of the server in" +
+            " the targets: ";
+    public static final String STATEMENT_FOR_FAILOVDER_RECONNECT = "Maximum retry attempts but couldn't connect " +
+            "to the one of the server in the targets: ";
+    public static final String STATEMENT = "Reconnect attempt not made because of close initiated by the server: ";
+    public static final String STATEMENT_FOR_BACK_OF_FACTOR = "The decay's value set for the configuration needs " +
+            "to be greater than -1. ";
+    public static final String STATEMENT_FOR_MAX_INTERVAL = "The maxInterval's value set for the configuration" +
+            " needs to be greater than -1. ";
+    public static final String STATEMENT_FOR_MAX_ATTEPTS = "The maximum doReconnect attempt's value set for the" +
+            " configuration needs to be greater than -1. ";
+    public static final String STATEMENT_FOR_INTEVAL = "The interval's value set for the configuration needs to be " +
+            "greater than -1. ";
+    public static final String STATEMENT_FOR_FAILOVER_INTERVAL = "The maxInterval's value set for the configuration " +
+            "needs to be greater than -1. ";
+
```

Comment:
If the constants are used in one location let us please declare them in the relevant class. If it is used only once there is no need to have them as a constant. For example I note `STATEMENT_FOR_FAILOVER_INTERVAL` need not be a constant. Check on other constants also.

Reply:
Repeating: _If the constants are used in one location let us please declare them in the relevant class.If it is used only once there is no need to have them as a constant._
Shall we fix for all constants

Reply:
I already moved those constant in the relevant class.
Please find https://github.com/ballerina-platform/ballerina-lang/pull/19054/files#diff-91a403e3e8df09f763a03646fd94b361R85